components/Header.tsx

```diff
@@ -98,7 +98,9 @@ const Header: React.FC<Props> = ({
     };
 
     // const isApiRef = router.query.slug && router.query.slug[0] === 'api-reference';
-    const isNonApiRef = router.query.slug && router.query.slug[0] === 'server-side';
+    const isNonApiRef =
```

Comment:
should we avoid using string literals here and replace with variables that refer to constants? asking because we might want to change the name for concepts and wouldn't want to have this get left out

Reply:
Yeah, planning on doing that